### DISCUSSION TITLE
Binary Size Increased

### DISCUSSION THREAD

**Discussion by vkbsb** [PhilipK/BoilR#303] [Ideas]
The linux binary size seems to have increased from 4MB to 16MB right now in the latest release. Is there a specific library thats' causing this? would be awesome to keep it at the original size :D

**Comment by PhilipK** [comment]
The problem is that I had to statically include ooenssl since it has been
removed on steam os.
This in itself is not a big size increase only a couple of kb.
But for some reason upx crashes when it tries to compress the binary on
Linux now.

If I compress it locally it's all fine, but if I do it on the github build
machines it just crashes.

So for now I disabled it again.
You can see that there are a couple of builds that have failed in the
github actions.

**Comment by PhilipK** [comment]
Manually compressed the latest version and re-uploaded it, about 5 mb now. I will have to find a way to get around this manual step in the future, but for now it works.

**Comment by vkbsb** [comment]
thank you this helps a lot :D
